test/scheduler.test.ts

```diff
@@ -109,9 +113,26 @@ describe("LocalPromiseStore", () => {
 
     // Expecting 1 schedules based on the search criteria
     let schedules: Schedule[] = [];
-    for await (const searchResults of store.schedules.search("schedule-1", { category: "search testing" })) {
+    for await (const searchResults of store.schedules.search("schedule-2", { category: "search testing" })) {
       schedules = schedules.concat(searchResults);
     }
     expect(schedules.length).toBe(1);
   });
+
+  test("search schedule with non-existing ID", async () => {
+    let schedules: Schedule[] = [];
+    for await (const searchResults of store.schedules.search("non-existing-id", {})) {
+      schedules = schedules.concat(searchResults);
+    }
+    expect(schedules.length).toBe(0);
+  });
+
+  test("delete non-existing schedule", async () => {
+    if (useDurable) {
```

Comment:
The test should be the same regardless of usedurable, if it's not we need to change one to match the other

Reply:
fixed.